notes/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from uuid import uuid4
+
+# Create your models here.
+
+class Note(models.Model):
+    title = models.CharField(max_length=200)
+    content = models.TextField(blank=True)
+    T_or_F = models.BooleanField(blank=True)
```

Comment:
On line 9, what is the function of the T_or_F variable? You might want to consider renaming it to something more intuitive for readability.